src/tests/test_util.py

```diff
@@ -130,6 +133,30 @@ def test_get_2of2_selections(self, *args):
 
         self.assertEqual(get_selection(roles), roles[1])
 
+    @patch('builtins.input', return_value=3)
+    @patch('sys.stdout', new=StringIO())
+    def test_get_bad_numeric_selection(self, *args):
+        """ Invalid numeric selection of two roles """
+        roles = [
```

Comment:
The release should not include real AWS account numbers, even in the test code. This is particularly undesirable given that this is a public release.